manifests/init.pp

```diff
@@ -143,7 +143,7 @@
   Variant[Array[String], String]  $root_mail_recipient = 'nobody',      # root_mail_recipient
   Optional[Boolean]               $chroot              = undef,
   Boolean                         $satellite           = false,
-  String                          $smtp_listen         = '127.0.0.1',   # postfix_smtp_listen
+  Variant[Array[String], String]  $smtp_listen         = '127.0.0.1',   # postfix_smtp_listen
```

Comment:
That would be great, though you would still want to accept `'all'`. It has been awhile since I've been deep in postfix configs though I thought you might be able to use other abbreviated strings besides hosts.